src/frontend/apps/drive/src/features/explorer/components/ExplorerFilters.tsx

```diff
@@ -41,6 +31,23 @@ export const ExplorerFilters = () => {
           </div>
         ),
         value: "file",
+        showSeparator: true,
+      },
+      {
+        label: t("explorer.filters.type.options.reset"),
```

Comment:
would be great to have smth like this built in the ui-kit

Reply:
yes